examples/nextjs/src/lib/axiom/client.ts

```diff
@@ -0,0 +1,12 @@
+import { Logger } from '@axiomhq/logging';
+import { AxiomProxyTransport, ConsoleTransport } from '@axiomhq/logging/transports';
```

Comment:
would it be easier if we simplify this import to:

```
import { Logger, AxiomProxyTransport, ConsoleTransport } from '@axiomhq/logging';
```

Reply:
@bdsqqq I agree to the concept, and that's what we are trying to do with the package, but the transports plays a core value (main funcationality) for the logging package and there is like 2 or 3 of them maximum. The Ai ones makes sense because they are different providers that would be implemented/developed separately as apposite to the transports here. wdyt?